cli/main.go

```diff
@@ -0,0 +1,46 @@
+//Package cmd implement a cli tool to sort, filter
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io"
+	"log"
+	"os"
+
+	"github.com/mh-cbon/semver/cli/cmd"
+)
+
+var version = "0.0.0"
```

Comment:
The version string here is to update on build with ldfflags, later it is bind to `-version` flag. It helps to remove manual ops on release. This value is only a default, could be empty whatever. Taking a finalized cli example, see https://github.com/mh-cbon/emd/blob/master/main.go#L19 and https://github.com/mh-cbon/emd/blob/master/.travis.yml#L30.